js/colorfunctions.js

```diff
@@ -60,3 +60,6 @@ return d > 10000 ? '#016c59' :
   d > null ? '#808080' :
   '#808080';
 }
+
```

Comment:
Like I mentioned earlier, please make sure any changes introduced are relevant to the overall change being requested for.